tests/security/pam/pam_login.pm

```diff
@@ -18,15 +18,20 @@ sub run {
     # Define the user and password, which are already configured in previous milestone
     my $user = 'suse';
     my $passwd = 'susetesting';
+    my $pam_sshd_tw = '';
     my $pam_login_tw = '';
 
     # Modify the login/sshd files to set the PAM authentication
     my $deny_user_file = '/etc/deniedusers';
     my $pam_sshd = '/etc/pam.d/sshd';
     my $pam_sshd_bak = '/tmp/sshd_bak';
     my $pam_login_bak = '/tmp/login_bak';
-    my $pam_sshd_tw = '/usr/etc/pam.d/sshd';
     my $pam_login = '/etc/pam.d/login';
+    if (is_sle || is_leap) {
+        $pam_sshd_tw = '/usr/etc/pam.d/sshd';
```

Comment:
Minor comment, I would suggest using a different variable name like `$pam_sshd_new` :)

Reply:
I just kept the old name in order to have consistency, as it is still the same variable!